src/test/java/io/supertokens/test/emailpassword/UpdateUsersEmailAndPasswordTest.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
- Need to add API test as well
- Add a test to update info of a user if they don't exist - should return status: "UNKNOWN_USER_ID"